src/validation/areChangeFilesDeleted.ts

```diff
@@ -17,12 +17,12 @@ export function areChangeFilesDeleted(options: BeachballOptions): boolean {
     process.exit(1);
   }
 
-  console.log(`Checking for deleted change files against "${branch}"`);
+  console.log(`Checking for deleted and renamed change files against "${branch}"`);
```

Comment:
I think it would make sense to rename the function too: `areChangeFilesDeletedOrRenamed`. A bit on verbose side, but reflects the purpose of the function :)

Reply:
I suppose that would be more descriptive. However, it will break the API. There are a couple of options I can see to avoid this:
1. Create a new more descriptive function while keeping the current one and marking it for deprecation in the future.
2. Create a new method that just checks for renamed change files and calling it based on a flag like `--disallow-renamed-change-files` which can be used in conjunction with the current function.
Which option should we go with - or are there other options?

Reply:
I think option #2 makes more sense :)

Reply:
@LarryKiniu This is an internal API, so you don't need to worry about breaking it.